src/pkg/packager/create.go

```diff
@@ -142,7 +142,7 @@ func (p *Packager) pullImages(imgList []string, path string) (map[name.Tag]v1.Im
 
 		pulledImages, err = imgConfig.PullAll()
 
```

Comment:
nit; this empty line could be removed

Reply:
I think I will merge as is.  Maybe I am weird but I like newlines between blocks (we should talk about style though more broadly though)